eng/pipelines/official.yml

```diff
@@ -134,6 +134,26 @@ extends:
                 inputs:
                   artifactName: Artifacts_Windows_NT_Release
                   targetPath: $(Pipeline.Workspace)/Artifacts_Windows_NT_Release
-              - template: /eng/pipelines/templates/steps/workload-publish.yml@self
-                parameters:
-                  feedForPublishing: ${{ parameters.feedForPublishing }}
\ No newline at end of file
+              - task: 1ES.PublishNuget@1
+                condition: ${{ ne(parameters.feedForPublishing, '') }}
+                displayName: 🟣 Publish package to AzDO
+                inputs:
+                  useDotNetTask: true
+                  packagesToPush: $(Pipeline.Workspace)/Artifacts_Windows_NT_Release/packages/Release/Shipping/*.nupkg
+                  packageParentPath: $(Pipeline.Workspace)/Artifacts_Windows_NT_Release/
+                  publishVstsFeed: ${{ parameters.feedForPublishing }}
+                  nuGetFeedType: internal
+                  allowPackageConflicts: false
+                  publishPackageMetadata: true
+              - task: 1ES.PublishNuget@1
+                condition: ${{ eq(parameters.feedForPublishing, '') }}
+                displayName: 🟣 Publish package to NuGet.org
+                inputs:
+                  useDotNetTask: true
+                  packagesToPush: $(Pipeline.Workspace)/Artifacts_Windows_NT_Release/packages/Release/Shipping/*.nupkg
+                  packageParentPath: $(Pipeline.Workspace)/Artifacts_Windows_NT_Release/
+                  publishVstsFeed: https://api.nuget.org/v3/index.json
+                  nuGetFeedType: external
+                  allowPackageConflicts: false
+                  publishPackageMetadata: true
+                  publishFeedCredentials: DotNet-SDK-Workloads-NuGet
```

Comment:
TODO: We need to create a [Service Connection](https://learn.microsoft.com/en-us/azure/devops/pipelines/artifacts/publish-public-registry?view=azure-devops&tabs=yaml#create-a-service-connection) to NuGet with an API Key with push access